docs/tutorial/installation.md

```diff
@@ -45,12 +52,30 @@ installation errors.
 In almost all cases, these errors are the result of network problems and not 
 actual issues with the `electron` npm package. Errors like `ELIFECYCLE`, 
 `EAI_AGAIN`, `ECONNRESET`, and `ETIMEDOUT` are all indications of such 
-network problems.  The best resolution is to try switching networks, or 
+network problems. The best resolution is to try switching networks, or 
 just wait a bit and try installing again.
 
 You can also attempt to download Electron directly from 
 [electron/electron/releases](https://github.com/electron/electron/releases) 
 if installing via `npm` is failing.
 
+If installation fails with an `EACCESS` error you may have to run it again with `sudo`:
```

Comment:
I think we should recommend fixing permissions on the global module folder over using sudo.  The global module directory should be read/writeable by all users